src/utils/Utils.tsx

```diff
@@ -882,4 +890,49 @@ export default class Utils {
   private static getDeviceLanguage(): string {
     return Utils.getLanguageFromLocale(Utils.getDeviceLocale());
   }
+
+  public static markerStatusChargingStations(connectorTab: Connector[], inactive: boolean) {
+    if (inactive) {
+      return charging_unavailable;
+    } else if (connectorTab.find((connector) => connector.status === ChargePointStatus.AVAILABLE)) {
+      return charging_available;
+    } else if (
+      connectorTab.find((connector) => connector.status === ChargePointStatus.FINISHING) ||
+      connectorTab.find((connector) => connector.status === ChargePointStatus.PREPARING)
+    ) {
+      return charging_finishing;
+    } else if (
+      connectorTab.find((connector) => connector.status === ChargePointStatus.CHARGING) ||
+      connectorTab.find((connector) => connector.status === ChargePointStatus.OCCUPIED)
+    ) {
+      return charging_charging;
+    } else if (
+      connectorTab.find((connector) => connector.status === ChargePointStatus.SUSPENDED_EVSE) ||
+      connectorTab.find((connector) => connector.status === ChargePointStatus.SUSPENDED_EV)
+    ) {
+      return charging_suspended;
+    } else if (connectorTab.find((connector) => connector.status === ChargePointStatus.FAULTED)) {
+      return charging_faulted;
+    }
+  }
+
+  public static markerStatusSiteArea(siteArea: SiteArea) {
+    if (siteArea.connectorStats.availableConnectors > 0) {
+      return charging_available;
+    } else if (siteArea.connectorStats.chargingConnectors > 0) {
+      return charging_charging;
+    } else if (siteArea.connectorStats.unavailableConnectors > 0) {
+      return charging_unavailable;
+    }
+  }
+
+  public static markerStatusSite(site: Site) {
+    if (site.connectorStats.availableConnectors > 0) {
+      return charging_available;
+    } else if (site.connectorStats.chargingConnectors > 0) {
+      return charging_charging;
+    } else if (site.connectorStats.unavailableConnectors > 0) {
+      return charging_unavailable;
+    }
+  }
```

Comment:
The methods are identical. I think you can pass a parameter in a single method to do both cases